flexmeasures/data/schemas/generic_assets.py

```diff
@@ -109,6 +114,38 @@ def _standardize_item(self, item) -> dict:
                 "Invalid item type in 'sensors_to_show'. Expected int, list, or dict."
             )
 
+    def flatten(self, nested_list) -> list:
+        """
+        Flatten a nested list of sensors or sensor dictionaries into a unique list of sensor IDs.
+
+        This method processes the following formats:
```

Comment:
More precise: It processes the following formats, for each of the entries of the nested list: